src/src/components/SideBar/SideBar.tsx

```diff
@@ -27,7 +26,8 @@ function SideBar(): React.FunctionComponentElement<React.ReactNode> {
   const [version, setVersion] = React.useState('unknown');
 
   useEffect(() => {
-    fetch('/version').then((response) => {
+    const baseURL = isDEVModeOn ? 'http://localhost:5000' : '';
```

Comment:
what do you think yo organise this into `helper` function which will give use exact `baseURL` value? so you can re use it in other places?

Reply:
introduced in https://github.com/G-Research/fasttrackml-ui-aim/pull/10/commits/a365332a3c8fa81211b26aeaa4c8a869fc7b3245